warp/warp.cabal

```diff
@@ -34,6 +34,14 @@ flag network-bytestring
 flag allow-sendfilefd
     description: Allow use of sendfileFd (not available on GNU/kFreeBSD)
 
+flag include-warp-version
+    description:
+        Exposes the version in Network.Wai.Handler.Warp.warpVersion.
+        This can cause a large runtime dependency closure due to depending on
+        files shipped with GHC itself.
```

Comment:
I think this is a Nix-specific concern, and should be flagged as such lest we spook non-Nix-using people.